README.rst

```diff
@@ -49,12 +49,19 @@ The -L flag can be used to whitelist certain words that are comma-separated plac
 
     codespell -S, --skip=
 
-Comma-separated list of files to skip. It accepts globs as well.  Examples:
+The -S (--skip) flag is used to skip over a comma-separated list of files and/or directories. It accepts globs as well. **Important note:** Do not append a '/' to a directory you want to skip. This currently renders the entry ineffective.
```

Comment:
"This currently renders the entry ineffective." does this just affect that one entry in the list, or the whole list? It would help to clarify either way, perhaps something like "This renders the entry for that particular directory ineffective". I'd suggest the currently is redundant, if it's intended to fix it, linking to the issue might be more useful. Either way it's irrelevant unless someone remembers to update the docs, if you want to leave it in, adding a version number and/or date would be more helpful. Both your example directories begin with ./, if that's also required it's not mentioned and I'd suggest is much more important.

Reply:
I've changed it based on your advice to `This renders the entry for that particular directory ineffective`.

re: the `./` in the examples.  
This is a quirk that I utilize in orrder to benefit form bash auto-completion but I'm not quite sure explaining that would be helpful as it would make this section overly-verbose. So I guess, removing this would be more appropriate.